src/vs/editor/common/textModelEvents.ts

```diff
@@ -361,6 +366,19 @@ export class ModelInjectedTextChangedEvent {
 	}
 }
 
+/**
+ * An event describing a change in special line height.
+ * @internal
+ */
+export class ModelSpecialLineHeightChangedEvent {
+
+	public readonly changes: ModelRawLineChanged[];
```

Comment:
Let's transport the changes with a new data type, there is no need to reuse/enlarge ModelRawLineChanged and it will just confuse all users of the event.